forc/src/cli/help.rs

```diff
@@ -0,0 +1,298 @@
+                        let mut current_word = String::new();
+                        loop {
+                            match chars.peek() {
+                                Some(c) => {
+                                    if *c == end_character {
+                                        let _ = chars.next();
+                                        args.push(current_word);
+                                        break;
+                                    } else if *c == '\\' {
+                                        let _ = chars.next();
+                                        if let Some(c) = chars.next() {
+                                            current_word.push(c);
+                                        }
+                                    } else {
+                                        current_word.push(*c);
+                                        chars.next();
+                                    }
+                                }
+                                None => {
+                                    break;
+                                }
+                            }
+                        }
+                    }
+                    c => {
+                        let mut current_word = c.to_string();
+                        loop {
+                            match chars.peek() {
+                                Some(' ') | Some('\t') | Some('\n') | Some('=') | Some('\'')
+                                | Some('"') | None => {
+                                    args.push(current_word);
+                                    break;
+                                }
+                                Some(c) => {
+                                    current_word.push(*c);
+                                    chars.next();
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+
+            args
+        }
+
+        fn help() -> &'static str {
+            Box::leak(format!("EXAMPLES:\n{}", examples()).into_boxed_str())
+        }
+
+        pub fn examples() -> &'static str {
+            Box::leak( [
+            $(
+            $crate::paste::paste! {
+                    format!("  #{}\n  forc {} {}\n\n", stringify!($($description)*), stringify!($command), $args )
+            },
+            )*
+            ].concat().into_boxed_str())
+        }
+    }
+}
+
+#[macro_export]
+macro_rules! cli_examples {
+    ($( [ $($description:ident)* => $command:tt $($arg:expr)* ] )*) => {
+            #[cfg(test)]
+            use $crate::serial_test;
+            $(
+            $crate::paste::paste! {
+                #[cfg(test)]
+                #[test]
+                #[serial_test::serial]
+                fn [<$($description:lower _)*:snake example>] () {
+                    let mut proc = std::process::Command::new("cargo");
+                    proc.arg("run");
```

Comment:
Command also has an `.args()` which allows you to chain once with multiple arguments.

Reply:
I also don't recall the context but I sometimes have problems with parsing arguments and using `proc.env("IFS", "''")` fixes that at times.

If you haven't done already I suggest a quick search to see if this is a good idea and if it may improve stability.

Reply:
I only recall that I had some issues a while back with `IFS` because of strings and spaces or something so I don't remember enough to provide any examples. If you're not seeing any issues then there's no reason to add this.